ntfysh_client/MainForm.Designer.cs

```diff
@@ -29,209 +29,219 @@ protected override void Dispose(bool disposing)
+            removeSelectedTopics.TabIndex = 0;
+            removeSelectedTopics.Text = "Remove selected";
+            removeSelectedTopics.UseVisualStyleBackColor = true;
+            removeSelectedTopics.Click += removeSelectedTopics_Click;
             // 
             // notificationTopics
             // 
-            this.notificationTopics.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left) 
-            | System.Windows.Forms.AnchorStyles.Right)));
-            this.notificationTopics.FormattingEnabled = true;
-            this.notificationTopics.ItemHeight = 15;
-            this.notificationTopics.Location = new System.Drawing.Point(13, 46);
-            this.notificationTopics.Margin = new System.Windows.Forms.Padding(4, 3, 4, 3);
-            this.notificationTopics.Name = "notificationTopics";
-            this.notificationTopics.SelectionMode = System.Windows.Forms.SelectionMode.MultiExtended;
-            this.notificationTopics.Size = new System.Drawing.Size(386, 199);
-            this.notificationTopics.TabIndex = 3;
-            this.notificationTopics.Click += new System.EventHandler(this.notificationTopics_Click);
-            this.notificationTopics.SelectedValueChanged += new System.EventHandler(this.notificationTopics_SelectedValueChanged);
+            notificationTopics.Anchor = System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left | System.Windows.Forms.AnchorStyles.Right;
+            notificationTopics.ContextMenuStrip = topicContextMenu;
+            notificationTopics.FormattingEnabled = true;
+            notificationTopics.ItemHeight = 15;
+            notificationTopics.Location = new System.Drawing.Point(13, 46);
+            notificationTopics.Margin = new System.Windows.Forms.Padding(4, 3, 4, 3);
+            notificationTopics.Name = "notificationTopics";
+            notificationTopics.SelectionMode = System.Windows.Forms.SelectionMode.MultiExtended;
+            notificationTopics.Size = new System.Drawing.Size(386, 199);
+            notificationTopics.TabIndex = 3;
+            toolTip.SetToolTip(notificationTopics, "Double click topic to send message");
+            notificationTopics.Click += notificationTopics_Click;
+            notificationTopics.SelectedValueChanged += notificationTopics_SelectedValueChanged;
+            notificationTopics.MouseDoubleClick += notificationTopics_MouseDoubleClick;
+            // 
+            // topicContextMenu
+            // 
+            topicContextMenu.Items.AddRange(new System.Windows.Forms.ToolStripItem[] { sendNotificationMenuItem });
+            topicContextMenu.Name = "topicContextMenu";
+            topicContextMenu.Size = new System.Drawing.Size(167, 26);
+            // 
+            // sendNotificationMenuItem
+            // 
+            sendNotificationMenuItem.Name = "sendNotificationMenuItem";
```

Comment:
Nice-to-have: It'd be good if an icon was found from the VS Icon Pack and placed against this menu item the same as all other menu items